app/assets/stylesheets/templates/_step.scss

```diff
@@ -6,4 +6,15 @@
     content: "";
     display: block;
   }
+
+  .form-card {
+    max-width: 72rem;
+
+    @include media($tablet-up) {
+      margin: {
+        left: auto;
+        right: auto;
+      }
+    }
+  }
```

Comment:
You mean create a new class named `max-width` or something and having these styles on that? 

I am totally down with that. In general, my only hesitation is that this is exactly how GCF does their max width and I'd like to copy the as much as possible: https://github.com/codeforamerica/gcf-backend/blob/master/app/assets/stylesheets/templates/_question.scss

what do you think?